lms/djangoapps/badges/backends/badgr.py

```diff
@@ -173,7 +173,8 @@ def _create_assertion(self, badge_class, user, evidence_url):
                 {
                     "url": evidence_url
                 }
-            ]
+            ],
+            "notify": settings.BADGR_ENABLE_NOTIFICATIONS
```

Comment:
```suggestion
            "notify": settings.BADGR_ENABLE_NOTIFICATIONS,
```
If you add trailing commas, then future diffs will be cleaner.

Reply:
Thanks, done! :)